### PR TITLE
Issue/595 editor tutorial

### PR DESCRIPTION
**Pull request by qwertynerd97**
Adds a tutorial document that walks users through the various aspects of the OboEditor.  The tutorial is automatically created as a new draft in the user's draft list if they have no existing drafts.  A user can also click the OboEditor Tutorial button to create a tutorial draft.

Resolves https://github.com/ucfopen/Obojobo/issues/595

### CODE REVIEW

File: packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json [hunk 1]
```diff
@@ -0,0 +1,8789 @@
...
+												}
+											]
+										}
+									},
+									{
+										"data": {
+											"indent": 0
+										},
+										"text": {
+											"value": "Try it! Use Tab and Shift+Tab, or the toolbar buttons, to increase and decrease the levels of this list.",
```
[the remark this answers is not in the document]
Comment (qwertynerd97) [reply packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json]: This appears to be an issue with the way lists are loaded back and forth between Obojobo and Slate.  I'll see if I can track down the error, and fix it in another pull request.

File: packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json [hunk 2]
```diff
@@ -7918,32 +8034,46 @@
...
 										"text": {
-											"value": "The review type describes how the student will see their answers after the attempt is over.  Selecting never ensures that, after every attempt, the student will only see which questions they got wrong, but will not be able to see the correct answers.  Selecting always allows the student to see what they got wrong and what the correct answer was after every attempt.  A value of no-attempts-remaining allows the student to see which question they got wrong after every attempt, but the review does not show them the correct answers until after they have exhausted all of their attempts.",
+											"value": "The review type describes how answers will be displayed after an attempt has ended.",
+											"styleList": []
+										}
+									},
+									{
+										"data": {},
+										"text": {
+											"value": "A value of never means that the student will only see whether or not they got a question wrong after every attempt. Correct answers will not be shown.",
```
Comment (gitnix) [inline packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json]: A value of never means that for every attempt, correct answers will never be shown to the student. The student will only see whether or not their response is incorrect.

File: packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json [hunk 3]
```diff
@@ -7918,32 +8034,46 @@
...
-												},
+													"start": 11
+												}
+											]
+										}
+									},
+									{
+										"data": {},
+										"text": {
+											"value": "A value of always means that the student will see both incorrect responses and the correct answers after every attempt.",
```
Comment (gitnix) [inline packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json]: A value of always means that for every attempt, the student will always see the correct answers.

File: packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json [hunk 4]
```diff
@@ -7918,32 +8034,46 @@
...
-													"start": 380
+													"start": 11
 												}
 											]
 										}
+									},
+									{
+										"data": {},
+										"text": {
+											"value": "A value of no-attempts-remaining means that the student will see whether or not they got a question wrong after every attempt. Correct answers are shown after the final attempt.",
```
Comment (gitnix) [inline packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json]: A value of no-attempts-remaining means that the student will only see correct answers on their final attempt. On all other attempts, only incorrect responses will be shown.

File: packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json [hunk 5]
```diff
@@ -0,0 +1,6970 @@
...
...47Q0XqSa5JK2dknSXuPm2qkC91ybs7dp2mOKHJJnZpp0NRo+y4cCspS6od9DdYa9Si4PpmSNWfAhKGq4TU48GjDVjaP0rRWlmVwbshwi806xvGYX1mx7dp7Uv08GuaPMs2ldwrAmKwhILI2+1ilTdUPsieJ2DtK59p1lo6UtR9Pr0LjxZ4Rliq1W1LQ8Ej7znHCS4hsNz19kL4x6WrqxlrPkuLfd30/s6Oqsx2sOcAS4E4ADY1o2DIYLBSRo/Q9JW0KaVhe5uNV7WOqHaKY9YsHDAnOOC+gjsT09jk4/uat+7nQtKdz+h4Wi26S3YcF48naTO6DL2EG9EScRGc4HsiVnq1VkydXZ9vRehaxL+rbfaIOESJnWD4ZKYaeptKvTjxXMxnK6PraJ6PVG1GVHtuta69BImRqwG9et4f4br72OpqKkvicmtjR6i7K+o5GHMzVArRlIx1nLSKMJyMhetKOd6hnaFoZtjtaiiXJscBMlsdrUhFmtSZoigCk0RQBItBASKTLOfOwDgpSo1c76EyqM2wtCTGmUaEi0ygUlpjBIaY7VJaZ8K26P6mlWewOe58BrWtOAvA7Neocl4UvDvLx1dSNycrpJck3Z0wmpNI8M6ykEAzJOIjGSY1ZzhC+fuUpNVx7ep6MpJcj7lTotaQ4tuhwB+8HCCM4Jnshdut4XtEG6jaXW1/wCmK1otcWZ6NjhxgYRJOWIAPMjmvHycoNrp9BXaaN+iCW16Zgg3g05G96pnmurw3WentUGurS+PA5tSHA9svvjlsCBWEBJjR1SkCIcARkYKmSjJU1ZabXIL6QLS0j1SII3ZKZwjOLhJcHwGpNOz5Vl6O02vLj6w9lp1Dj7y8jR8F0NPUc3xXRPp7+/5dmr1m1SPtFeuZ2ed0t0Ro1fWp/6Tv7RLTxb5QvP1/DdHUX6f0v0+xvHaGuZlo9DWsAIqFz5GJ9VoGMkASSe1cOr4LenjGXG+b5V7i3tOXNHodH2QUWXGkmTLjmfJensWww2XTxjxfV9zmlqNso8rtSM2yL1aM2zPUVIylIx1pWsTmnJmQhWYWBoVA2OGoJscNQTZRjUhrmXDVJsMGpDHAQWjoSGNCB2ddQAwakUO0KSkx4SKsISY0xwkWmMEh2Zxo2l1nXXB1nvZbwNQO9YeX0t5vMVl3Nd5Kqs2ArVqyVIxWbRVNnWYE9YC07gTMDu
```
Comment (gitnix) [inline packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json]: A value of never means that after every attempt, the student will only be able to see if an answered question is incorrect.

Comment (gitnix) [inline packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json]: could probably use a list here for the required nodes

**Comment by gitnix** [comment]
Can't seem to scroll through the side menu on viewer or editor.

**Comment by zachberry** [comment]
@gitnix Good catch, made a separate issue - #709

**Pull request merged**